classes/invitation/invitations/CustomInvite.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @file invitation/invitations/CustomInvite.php
+ *
+ * Copyright (c) 2023 Simon Fraser University
+ * Copyright (c) 2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class CustomInvite
+ *
+ * @ingroup invitations
+ *
+ * @brief CustomInvite
+ */
+
+namespace PKP\invitation\invitations;
+
+use Illuminate\Mail\Mailable;
+
+class CustomInvite extends BaseInvitation
```

Comment:
Is this just an example, or will this class be used in production?

Reply:
Just a showcase - not intended to include it in the final version of the PRs